### PR TITLE
Buffs laser damage from 20 to 25 burn damage.

### PR DESCRIPTION
**Pull request by CPTANT**
The laser gun is basically one of the least imposing weapons on the station. Someone throwing glass shards is almost more menacing than someone with a laser gun.

Basically he only thing its good at vs players is shooting already downed targets. 

This makes the laser gun a bit more viable in humanoid vs humanoid combat and makes it look less like a glorified flashlight. 
:cl: 
Buffs laser rifle damage from 20 to 25 burn damage
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
Can you give it more ammo too? 14 egun 16 laser rifle?

**Comment by WJohn** [comment]
This is actually a pretty major balance change. I think it may let you one shot blob pieces, and it lets you two shot xeno drones.

**Comment by Supermichael777** [comment]
laser guns are deadly because of how hard it is to escape after you get taged a few times, not based on straight up damage. also buff the ammo or thedamage but not both.

**Comment by CPTANT** [comment]
Everything else that hits you makes it even more difficult to escape compared to lasers. Things like: tasers, disablers, bullets, thrown spears, pneumatic cannons.

Lasers are downright at the bottom of imposing weapon.

**Comment by WJohn** [comment]
You realize pure murdering is still never going to murder because stuns are *still* going to be more important. This is, unless of course, lasers becoming hideously, disgustingly strong. Frankly, they shouldn't.

**Comment by KorPhaeron** [comment]
>ranged, spammable, rechargable weapon 
>damage already outclasses almost every melee attack
>buff it

Why? And tasers will still be better, you'll just execute victims before they can type more.

**Comment by Steelpoint** [comment]
Welcome to why we will never get anywhere but nerfing things, since its easier to nerf than to buff.

Argumentations and all aside, I do agree in principle that a dedicated weapon, a actual laser, should be a deadly weapon. If it were up to me I would buff the damage to 30 and make it have a 20 rounds charge.

25 burn damage seems like a ok start for making the laser a actually viable offensive weapon.

If blob cores and the such prove too easy to kill then just slightly buff their health. Just be sure to do testing.

**Comment by CPTANT** [comment]
@KorPhaeron 
Spammable is dubious. An energy gun with 10 shots gives you enough damage to practically take ONE person to crit with a full charge. A laser gun with 12 shots gives you MAYBE enough charge to crit a second person.

Being rechargable is way more of a con than a pro. Ballistics with cheap ammo just lets you almost instantly change magazine and resume firing, and is in that way is way more spammable. 

lets compare the Syndicate SIDEARM pistol with the main laser gun.

The Stechkin:
has superior damage
has a superior damage type (less blocked by armour)
causes bleeding
fits into pockets
can be silenced
allows you to quickly change ammo and continue firing.

cons:
slightly smaller magazine size
needs ammo

I agree that that tasers and other stuns will still be better. Which is why we shouldn't needlessly punish lethal weapons. With the current system there has to be a balance between the two and this change makes it a bit less of a discouragement to go for the lethal options.

**Comment by Steelpoint** [comment]
I would take a Sketchin over a Laser Rifle any day of the week, the Pistol has such versatilely and deadliness to be very viable as a side arm.

Also can we stop spreading falsehoods with the current iteration of the auto rifles, are we all forgetting their ammo is expensive to get and cannot be produced easily or cheaply?

I would argue this weapon should be around the level of the most basic syndicate weapons, so a 25 damage, or 30, seems fine.

**Comment by Zilenan91** [comment]
We need to buff its ammo capacity. OR ADD LASER MAGAZINES WHICH HAS BEEN A PR FOR LIKE 9000 YEARS.

**Comment by Amemelius** [comment]
Good PR. Honestly, I'd also propose increasing the ammo capacity a little bit as well. It's still pretty bad even at 25.

**Comment by Steelpoint** [comment]
Baby steps, buffing directly to 35 damage is going to be harder than a small buff to 25 damage.

Also allows us to see how the change goes for a few weeks.

**Comment by Anonus** [comment]
This will make lasers absolutely wreck blobs, as they'll oneshot normal blob pieces.
This also buffs eguns and anything that uses the laser projectile.

**Comment by Metacide** [comment]
Seems like a pretty huge balance change and I don't see why it's necessary.

**Comment by Robustin** [comment]
Lasers are fucking fine and they do everything they promise to do without going overboard.

This is going overboard. 

Syndies are supposed to have superior lethals because they lack tasers.

**Comment by Lumipharon** [comment]
Lasers aren't designed for turbomurder.
They're good at killing shit like xenos, blobs and carp, for that sweet sweet double damage bonus.

**Comment by CPTANT** [comment]
turbomurder?

25 damage and 12 shots means you can under practical circumstances crit a max of 2 people.

**Comment by Fox-McCloud** [comment]
I think everyone is forgetting about the existence of the R&D Laser cannon, which fulfills this niche.

If no one is using it, in its current iteration, then reduce its w_class so it can fit in backpacks.

**Comment by Zilenan91** [comment]
The issue isn't that its weight class is too high, the issue is that it comes out of science, the department more concerned with making itself and only itself mechs rather than handing things out.

**Comment by Lumipharon** [comment]
Laser cannon used to a normal sized item.
I think pap or someone increased the size, not really sure why.

**Comment by GunHog** [comment]
>Laser cannon used to a normal sized item.
I think pap or someone increased the size, not really sure why.

I traced it to this PR: https://github.com/tgstation/-tg-station/pull/4427

**Comment by MactoPerFuror** [comment]
I feel this is the right way to go, just tweak blobs a touch and see how it works in-game. In theory this should not affect the game in any huge way. 

Also the R&D laser cannon alternative is laughable. Science is too unreliable.

**Comment by Dorsisdwarf** [comment]
Laser cannon will need a damage buff to be worth the investment of resources and huge asspain in getting it off the grubby mitts of the labrats

**Comment by CPTANT** [comment]
@Dorsisdwarf 
I feel the right way to go for the laser cannon would be a large increase in capacity, so ~doubling its ammo.

**Comment by Supermichael777** [comment]
>that time when you deeply consider balance memes to buff a under-powered gun just because of a pap nerf.

considering buffing death-squid rifles and laser cannons down to weight class 3 just because an enormous revolver and the literal fuck the universe gun just need the buff

**Comment by Robustin** [comment]
>Oh no my gun can only kill 2 people at a time

This fucks with xenos, blobs, shadowlings, and more. It disrupts fundamental game balance by pushing us back toward the autorifle period where sec just breaks out lethals as the first line of defense because lasers are better at "stunning" people than disablers.

Right now when some jerkoff with a stolen laser tries to ambush me, I have a fighting chance to escape, this change accelerates not only the injury slowdown that makes you an easy target for follow-up shots, but increases ammo efficiency relative to non-lethal options. There's also the issue of being able to call for help, certainly doable when being lasered now, far less likely at 25/shot. 

I've been using lasers 5ever and never though "geez these things just don't kill people fast enough"... they're ranged, they penetrate glass, they travel indefinitely, and they can be fired faster than any melee weapon... they're FINE.

**Comment by CPTANT** [comment]
A lasergun can't even kill 2 people at a time like you fakequote. It has a damage potential of 240, meaning you have enough damage to KILL one person. 

What chance to escape is there when you get hit by a taser? Or a disabler? Or any other type of stun. Like I said stun combat is still way more effective. This is simply not as needlessly punishing the lethal option.

**Comment by Anonus** [comment]
You need 100 damage to crit someone, and short of them being a ling or having a healing virus, that means you've killed them.

**Comment by KorPhaeron** [comment]
Yeah why the fuck do you keep acting like people have 200 health? You know you can just kick them after they're in crit right? Or leave them there.

**Comment by CPTANT** [comment]
There are way more situations than a healing virus that requires you to actually finish someone off. Like people with holoparasites, any team antag or crew with buddies around or lings. 

Holoparasites for one really highlight the lack of firepower in lasers. A holoparasite with a damage modifier of 0.8 can tank 13 shots before its host is down. that is a LOT of hits.

**Comment by GunHog** [comment]
Crit with laser, beat them with a melee object, or flog with the gun once the target is in crit. This is basic combat that even an unrobust scrub like me knows.

**Comment by Anonus** [comment]
>short of being a ling, having a healing virus, or being healed
>way more situations that require you to finish someone off

>13 shots before the host is dead
>11 shots on a holoparasite with 0.8 damage resist will kill the host

**Comment by CPTANT** [comment]
combat is more accurately described as "shoot taser once, cuff/bash them to crit" done. or shoot 3 disabler bolts and do the same.

**Comment by CPTANT** [comment]
would simply changing blob health to 26 (from 25) be a good change to fix lasers one-shotting blob tiles?

**Comment by Steelpoint** [comment]
Increasing the blob tile to have 26 or so health would be fine.

Here's a examination of the amount of laser rounds it would take to crit Aliens, both from 20 damage and 25 damage, accounting for the increased damage lasers deal.

-----
20 Damage Lasers

Drone - Three (3) shots to crit.

Hunter - Four (4) shots to crit.

Sentinel - Four (4) shots to crit. (Barely)

Praetorian - Seven (7) shots to crit.

Queen - Ten (10) shots to crit. (Barely)
-----

Versus

-----
25 Damage Lasers

Drone - Two (2) shots to crit. (Barely)

Hunter - Three (3) shots to crit. (Barely)

Sentinel - Three (3) shots to crit (Barely)

Praetorian - Five (5) shots to crit. (Barely)

Queen -  Eight (8) shots to crit. (Barely)
----

The barely affix means the damage dealt is just a few damage points off of requires another rounds to crit.

In conclusion, against aliens default Lasers, requires one to two less laser rounds to crit. The Praetorian and Queen can get one health point buff and requires only one more to crit if need be.

Also Laser Cannons still outdo Laser Rifles, with it effectivly being a two hit crit on most Aliens sans the Praetorian and Queen, requiring four and five cannon rounds respectively.

**Comment by Metacide** [comment]
But there isn't even a particularly good reason to do this. I've never noticed anyone complain about lasers and never found them an issue myself. Where did this idea come from? Shouldn't they just be as powerful as is reasonably balanced, as in how they are now? 

You can kill two people straight up and seriously injure three making them easy to finish off - that only on one charge. You can easily stockpile a backpack full of these. People murder swathes of people just fine using them.

This really just looks like change for the sake of change, and will result in a bunch of unnecessary busywork to rebalance other things like xenos and blobs. I can't see why you'd want to do this.

**Comment by KorPhaeron** [comment]
I really dont understand it either. He says its about humanoid combat but then cites parasites before promising to rebalance every non human entity in the game around the new damage.

Blobs take different damage from heat vs brute weapons, and buffing their health to counter lasers makes them stronger against everything else.

And it will still be trash for anything but tase+execute to anyone wearing armour.

**Comment by CPTANT** [comment]
@KorPhaeron 
Like I have said multiple times now, the major incentive is the complete superiority of stuns over lethal combat in basically every situation. With this change stuns will still be superior, but there is one less lethal option that is completely needlessly punished. 
You tell me with a straight face that someone with a laser gun (a dedicated weapon) is not completely outclassed by someone with a hybrid taser (a tool designed for non-lethally subduing suspects) in a fight that is to the death. 

Blobs having health set to 26 health would only affect weapons that do a multiple of 5 damage. There is nothing that makes sense to use against a blob that this effects except the laser.

One buff of 1 health to blobs <--> buffing everything.  The fuck are you smoking.

I would like to see the effect this has against for example the quoted aliens before deciding to apply buffs.

**Comment by Anonus** [comment]
So instead of going after the problem, stuns being stronger than lethals in 99% of situations, you buff lasers, wreck a bunch of stuff balanced around lasers, and STILL LEAVE STUNS BETTER THAN LASERS?

Also for what it's worth you're doing the blob buff wrong, you need to edit the health var to 26 as well.

**Comment by Lumipharon** [comment]
The solution to changing one item that utterly fucks balance in other areas of the game is to not make that change.
This is the exact trap that steel fell into with his rifles.

Add more ammo to lasers if you think they're UP, not more damage. Also enjoy your taser vs laser while you get lasered through an airlock or window.

**Comment by Dorsisdwarf** [comment]
#bring_back_egun_tasers

**Comment by Fox-McCloud** [comment]
Because taze and lase was so fun to play against.

**Comment by Steelpoint** [comment]
it worked for years until Paprika removed it, then suddenly people hated it.

**Comment by Fox-McCloud** [comment]
Sure, it "worked" just like I'm sure if the energy gun did damage and stunned at the same time, it would "work", but that doesn't necessary make that a *good* thing. 

There's zero interaction between a spammable stun weapon that can instantly be fellowed up with lethal shots. Person gets hit once and they're dead.

For all the problems stamina has, it still has far more interaction between the shooter and the person getting shot since it's not *effectively* one shot, one kill.

**Comment by Steelpoint** [comment]
Need I bring up the point that there's little functional difference between swapping from a Taser then to a Laser for the Stun + Kill?

Once again its more ammo efficient to use two separate guns for that wombo combo than it is to have it in a single gun.

**Comment by Dorsisdwarf** [comment]
Tase and lase was fine when you could juke taser shots at supersonic speed

**Comment by dumpdavidson** [comment]
Honestly you underestimate how prevalent lasers are in combat. They're very often used by the more robust traitors or desperate security in rev who are taking on crowds of people. Just because you don't use them as an opener to a fast moving and aware target doesn't mean they're irrelevant.
Also since the advent of adrenal implants/glands you're often well off using them.

That being said the difference between 4 and 5 shots to crit somebody is actually huge. I am pretty sure we used to have the classic laserguns being 25 or 30 damage before and it sucked ass.

**Comment by MrStonedOne** [comment]
Please make a changelog,

You can add it to the PR's body rather then as a file now, see here: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157

**Comment by duncathan** [comment]
Personally I'd rather see a significant increase in capacity so that you can shoot more

**Comment by Cheridan** [comment]
I think Anonus, Kor, Lumipharon etc have the right idea here. This doesn't weaken stuns, it just makes lasers even better as a stun followup while also messing up the balance for blobs/simple animals and others.

**Comment by Dorsisdwarf** [comment]
Reviewed!
